### DISCUSSION TITLE
Demo experiment produces "The experiment failed to load." message

### DISCUSSION THREAD

**Discussion by hamster789** [jspsych/jsPsych#2525] [Q&A]
Hi, I am very new to javascript and coding. I am currently trying out the demo experiment on JsPsych- simple reaction time task. When I try to run the experiment with Visual Studio Code and open it with a live sever, it shows that "The experiment failed to load." I am not sure why the experiment is not working as I directly copied the coding in the webpage on JsPsych. :(

**Comment by jodeleeuw** [comment]
Hi @hamster789,

This is probably because the image files used in the experiment are not downloaded. See [step 3](https://www.jspsych.org/7.1/tutorials/rt-task/#part-3-show-instructions) of the tutorial for a description of how to save the image files.

**Reply by hamster789** [reply]
It worked! Thanks a lot!!
